package-res/resources/web/pentaho/type/Context.js

```diff
@@ -57,24 +57,39 @@ define([
    * @name pentaho.type.Context
    * @class
    * @implements pentaho.type.IContext
-   * @classDesc The `Context` class contains _Value_ instance classes
+   * @classDesc A `Context` instance holds constructors/classes of _Value_ types
    * _configured_ for a particular _context_.
    *
+   * A context has several variables that define it:
+   * [container]{@link pentaho.type.Context#container},
+   * [user]{@link pentaho.type.Context#user},
+   * [theme]{@link pentaho.type.Context#theme} and
+   * [locale]{@link pentaho.type.Context#locale}.
+   * Context variables are specified at construction time, through same named keyword arguments.
+   *
+   * These variables are used to select applicable configuration rules
+   * that are applied to the types provided by the context.
+   *
+   * Because configuration rules rely on the identification of types,
+   * only non-anonymous types -
+   * those that have an [id]{@link pentaho.type.Value.Meta#id} -
+   * can be _directly_ configured.
+   *
```

Comment:
Yes, I noticed, but the level here is a bit too deep. I felt that a less technical and more motivational explanation was needed in order to capture the interest for this core concept and to give an idea of its systemic influence.
Mind you, I know it's hard to come up with such an explanation, I've been trying to come up with one myself to better illustrate my point...